test/modules/geo-layers/tile-layer/tile-layer.spec.js

```diff
@@ -69,7 +69,7 @@ test('TileLayer', async t => {
   const testCases = [
     {
       props: {
-        data: 'http://echo.jsontest.com/key/value'
+        data: 'http://echo.jsontest.com/type/Point'
```

Comment:
After adding the mimeTypes, [this test](https://github.com/visgl/deck.gl/blob/master/test/modules/geo-layers/tile-layer/tile-layer.spec.js#L71) was broken:

` [Error: GeoJSON does not have type]`

It passed the test but it was not working well. `load` launched an exception that was captured [here](https://github.com/visgl/deck.gl/blob/master/modules/geo-layers/src/tile-layer/tile-2d-header.js#L72) so the content of the tile was never set. 

This line + mimeTypes fixed it 💪